internal/netclassify/classify.go

```diff
@@ -36,6 +36,23 @@ var (
 	pythonSourceRegex = regexp.MustCompile(`^(?P<package>[\w\.]+)-(?P<version>.+?)(?P<ext>\.(zip|tar\.gz|tar\.bz2|tar\.xz|tar\.Z|tar))$`)
 )
 
+// NPM
+var (
+	npmAPIRegex  = regexp.MustCompile(`^https://registry\.(npmjs\.org|yarnpkg\.com)/((@[^/]+/)?[^/]+)/([^/]+)/?$`)
+	npmFileRegex = regexp.MustCompile(`^https://registry\.(npmjs\.org|yarnpkg\.com)/((@[^/]+/)?[^/]+)/-/([^/]+)-([^/-]+)\.tgz$`)
+)
+
+// Maven
+var (
+	mavenCentralRegex = regexp.MustCompile(`^https?://(repo1\.maven\.org/maven2|central\.maven\.org/maven2|maven-index://repo1\.maven\.org)/(.+)/([^/]+)/([^/]+)/([^/]+)$`)
```

Comment:
what's this matcher? `maven-index://repo1\.maven\.org`, not covered in tests

Reply:
yep missed that. removed.